pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -96,6 +96,22 @@ func classificationSupported(config *config.Config) bool {
 	return kprobe.ClassificationSupported(config)
 }
 
+// skipIfUsingNAT skips the test if we have a NAT rules applied.
+func skipIfUsingNAT(t *testing.T, ctx testContext) {
+	if ctx.targetAddress != ctx.serverAddress {
+		t.Skip("test is not supported when NAT is applied")
+	}
+}
+
+// composeSkips skips if one of the given filters is matched.
+func composeSkips(skippers ...func(t *testing.T, ctx testContext)) func(t *testing.T, ctx testContext) {
```

Comment:
that one sounds (and looks) generic, why is it linux only?

Reply:
linter complained about it is unused in the windows version